### PR TITLE
Add Kafka to circle setup

### PR DESCRIPTION
**Pull request by beckje01**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -2,11 +2,31 @@ machine:
...
+    # Start Zookeeper in the background
+    - '~/kafka_2.11-0.10.0.0/bin/zookeeper-server-start.sh ~/kafka_2.11-0.10.0.0/config/zookeeper.properties':
+        background: true
+
+    # Start Kafka in the background
+    - '~/kafka_2.11-0.10.0.0/bin/kafka-server-start.sh ~/kafka_2.11-0.10.0.0/config/server.properties':
+        background: true
+
+    # Wait for Kafka to start
+    - sleep 10
```
Comment (charliek) [inline circle.yml:24]: I'd pull in our script to wait for a socket so you don't have to guess on a wait time.

File: ratpack-kafka-producer/src/test/groovy/smartthings/ratpack/kafka/KafkaProducerServiceSpec.groovy [hunk 2]
```diff
@@ -10,14 +10,18 @@ import spock.lang.Specification
 
 class KafkaProducerServiceSpec extends Specification {
 
+	String getTestKafkaServers() {
+		return System.getenv("KAFKA_SERVER") ?: '192.168.99.100:9092'
```
Comment (charliek) [inline ratpack-kafka-producer/src/test/groovy/smartthings/ratpack/kafka/KafkaProducerServiceSpec.groovy:14]: Would it make sense to not have a default here at all and blow up if it isn't set?

Comment (beckje01) [inline ratpack-kafka-producer/src/test/groovy/smartthings/ratpack/kafka/KafkaProducerServiceSpec.groovy:14]: Potentially I was setting a default to work for local dev.

**Comment by charliek** [comment]
Both of my comments are probably something we can iterate on to keep this work moving, but I think would be good changes.

**Comment by beckje01** [comment]
I'll follow up with a PR for the socket wait.

**Pull request merged**